pkg/kubelet/kubelet_node_status.go

```diff
@@ -50,6 +50,12 @@ const (
 	// maxNamesPerImageInNodeStatus is max number of names per image stored in
 	// the node status.
 	maxNamesPerImageInNodeStatus = 5
+
+	//Label for storing provided IP addresses
+	LabelProvidedIPAddr = "beta.kubernetes.io/provided-node-ip"
```

Comment:
that lives in the staging/ tree, I think - no need to go through that complexity, I think.

I still don't quite grok what this is, or why it is a label.  What piece of kubelet is this preserving?

@dchen1107 for maybe a clue?